rockcraft.yaml

```diff
@@ -1,32 +1,40 @@
 name: snapcraft-core24
 base: ubuntu@24.04
-version: 8.4.1
-summary: easily create core24 snaps
+version: 8.9.2
+summary: Package, distribute, and update any app for Linux and IoT.
```

Comment:
We probably still want to include a mention that this specific rock is fore making core24 snaps.

Reply:
Good idea, I updated the description.